code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -212,3 +213,11 @@
 
 /obj/item/restraints/handcuffs/cable/zipties/used/attack()
 	return
+
+/obj/item/restraints/handcuffs/cable/twimsts
```

Comment:
Because of this path things like these would be inherited
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/df91089a-85c2-4463-8eae-fd42143988d2)

![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/ac932362-e963-4228-98eb-f75d94c6b44a)